pep-0514.txt

```diff
@@ -200,12 +240,16 @@ always named ``InstallPath``, and the default value must match ``sys.prefix``::
 If a string value named ``ExecutablePath`` exists, it must be a path to the
 ``python.exe`` (or equivalent) executable. Otherwise, the interpreter executable
 is assumed to be called ``python.exe`` and exist in the directory referenced by
-the default value.
+the ``(Default)`` value.
 
 If a string value named ``WindowedExecutablePath`` exists, it must be a path to
 the ``pythonw.exe`` (or equivalent) executable. Otherwise, the windowed
 interpreter executable is assumed to be called ``pythonw.exe`` and exist in the
-directory referenced by the default value.
+directory referenced by the ``(Default)`` value.
+
+Tools should be able to handle the case where the referenced executables do not
+actually exist, particularly when values for ``ExecutablePath`` and
+``WindowedExecutablePath`` are inferred from the ``(Default)`` value.
 
```

Comment:
I honestly don't understand what use there is for an installation that doesn't include information on how to start the Python interpreter. I guess it's true that there isn't always a ```python.exe``` - Jython, for example, has ``jython.jar```. But tools need *some* way of knowing how to start the Python interpreter.

Maybe the two values should be ```InterpreterCommand``` and ```WindowedInterpreterCommand```. If they are not present, then ```<InstallPath>\python.exe``` and ```<InstallPath>\pythonw.exe``` should be assumed. The windowed interpreter is optional, but the "normal" interpreter command *must* be available (either as python.exe or in an explicit form). Tools shouldn't be expected to deal with a registered interpreter that they can't invoke.

Do you have an example of a use case where you need to introspect the installed Pythons, but you never actually need to run the interpreter?